tests/ludwig/encoders/test_text_encoders.py

```diff
@@ -292,3 +289,42 @@ def test_tfidf_encoder(vocab_size: int):
     inputs = torch.randint(2, (batch_size, sequence_length)).to(DEVICE)
     outputs = text_encoder(inputs)
     assert outputs[ENCODER_OUTPUT].shape[1:] == text_encoder.output_shape
+
+
+def test_hf_auto_transformer_use_pretrained(tmpdir, csv_filename):
```

Comment:
nit: The `True` case would be tested implicitly elsewhere, correct? If not, maybe we could parametrize the test with both cases.